dll/win32/shell32/wine/shpolicy.c

```diff
@@ -856,6 +856,9 @@ DWORD WINAPI SHRestricted (RESTRICTIONS policy)
 	}
 
 	lstrcpyA(regstr, strRegistryPolicyA);
+#ifdef __REACTOS__
+    lstrcatA(regstr, "\\");
+#endif
```

Comment:
Since wine accepted the patch, this shouldn't be in a `#ifdef __REACTOS__` any more.